exonum_client/message.py

```diff
@@ -158,8 +159,10 @@ def from_hex(cls, message_hex: str, artifact_name: str, tx_name: str) -> Optiona
             exonum_message = cls(service_id, message_id, decoded_msg, prebuilt=exonum_msg.any_tx.SerializeToString())
 
             cls._set_signature_data(exonum_message, author, signature, bytes.fromhex(message_hex))
+            logging.debug("Exonum message parsed successfully.")
             return exonum_message
         except ProtobufDecodeError:
+            logging.error("Failed to parse an Exonum message.")
```

Comment:
I believe information about what kind of message we tried to parse will be more helpful than just a fact that we've failed.